src/drivers/esp32/src/esp_slip.c

```diff
@@ -42,3 +44,32 @@ static uint8_t generateChecksum(uint8_t *sendBuffer, esp_slip_send_packet *sende
   }
   return checksum;
 }
+
+static void assembleBuffer(uint8_t *sendBuffer, esp_slip_send_packet *senderPacket)
+{
+  sendSize = senderPacket->dataSize + ESP_OVERHEAD_LEN + 2;
+
+  sendBuffer[0] = 0xC0;
+  sendBuffer[1] = DIR_CMD;
+  sendBuffer[2] = senderPacket->command;
+  sendBuffer[3] = (uint8_t)((senderPacket->dataSize >> 0) & 0x000000FF);
+  sendBuffer[4] = (uint8_t)((senderPacket->dataSize >> 8) & 0x000000FF);
+
+  if (senderPacket->command == FLASH_DATA) // or MEM_DATA
```

Comment:
This comment confuses me more than it enlightens me, what does it mean?

Reply:
It refers to another "command" that requires a checksum to be sent to the ESP, but considering we will not use it, I will remove the comment